lib/workers/bin/run_single_worker.rb

```diff
@@ -57,14 +57,22 @@
 worker_class = worker_class.constantize
 worker_class.before_fork
```

Comment:
Side question: @NickLaMuro is this needed? ^  I don't remember why we need to call before_fork.